Aquality.Selenium.Core/src/Aquality.Selenium.Core/Configurations/VisualizationConfiguration.cs

```diff
@@ -24,5 +25,14 @@ public VisualizationConfiguration(ISettingsFile settingsFile)
         public int ComparisonWidth => settingsFile.GetValueOrDefault(".visualization.comparisonWidth", 16);
 
         public int ComparisonHeight => settingsFile.GetValueOrDefault(".visualization.comparisonHeight", 16);
+
+        public string PathToDumps
+        {
+            get
+            {
+                var pathInConfiguration = settingsFile.GetValueOrDefault(".visualization.pathToDumps", "../../../Resources/Dumps/");
```

Comment:
I guess, `Resources/Dumps` might be not clear enough to indicate its purpose. @mialeska, how about `Resources/VisualTesting/Dumps` or `Resources/VisualDumps`?

Reply:
Resources/VisualDumps works for me